site/layouts/partials/footer.html

```diff
@@ -44,5 +44,58 @@
     </script>
     {{ end }}
 
+    <script>
```

Comment:
@corneliusweig should we move these to a .js file at this point?

Reply:
Yes, that makes sense to me. We don't have to do it in this PR though. Maybe open a new issue? [[main reason: collaboration on the shared PR seems to be not straightforward]]

Reply:
ack